### PR TITLE
FPET-968 : COS PR to implement remove draft orders .

### PR DESCRIPTION
**Pull request by pragnesh-patel3**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `enable_keep_helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

FPET-968

### Change description ###

This is to remove the draft orders uploaded by mistake in a case.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by priteshkr — COMMENTED** [review]
Please give the proper headings in the PR and update the description with JIRA link properly.

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/RemoveDraftOrderController.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    private final ObjectMapper objectMapper;
+    private final RemoveDraftOrderService removeDraftOrderService;
+    private final AuthorisationService authorisationService;
+
+    public static final String DRAFT_ORDER_COLLECTION = "draftOrderCollection";
+    public static final String REMOVED_DRAFT_ORDER_TEXT = "removeDraftOrderText";
+    public static final String CONFIRMATION_HEADER = "# Draft order removed";
+    public static final String CONFIRMATION_BODY_FURTHER_DIRECTIONS = """
+        ### What happens next \n We will delete this order.""";
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/RemoveDraftOrderController.java:46]: Remove the unused

File: src/main/java/uk/gov/hmcts/reform/prl/services/RemoveDraftOrderService.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+    public Map<String, Object> getDraftOrderDynamicList(CaseData caseData,
+                                                        String eventId,
+                                                        String authorisation) {
+
+        Map<String, Object> caseDataMap = new HashMap<>();
+        List<Element<DraftOrder>> supportedDraftOrderList = new ArrayList<>();
+        caseData.getDraftOrderCollection().forEach(
+            draftOrderElement -> {
+                log.info(" ---> eventId {}", eventId);
```
Comment (priteshkr) [inline src/main/java/uk/gov/hmcts/reform/prl/services/RemoveDraftOrderService.java:42]: Remove this logger

Comment (pragnesh-patel3) [reply src/main/java/uk/gov/hmcts/reform/prl/services/RemoveDraftOrderService.java]: Removed.

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/RemoveDraftOrderController.java (hunk 1 above)
Comment (pragnesh-patel3) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/RemoveDraftOrderController.java]: Removed.

**Review by priteshkr — APPROVED** [review]

**Pull request merged**